tools/clang/test/HLSLFileCheck/hlsl/workgraph/called_function_arg_record_object.hlsl

```diff
@@ -13,7 +13,7 @@ void loadStressWorker(
     inout DispatchNodeInputRecord<loadStressRecord> inputData,
     GroupNodeOutputRecords<loadStressRecord> outRec)
 {
-    // CHECK: getelementptr inbounds %struct.loadStressRecord.0, %struct.loadStressRecord.0
+    // CHECK: getelementptr %struct.loadStressRecord.0, %struct.loadStressRecord.0
```

Comment:
Pretty sure this change is unnecessary if you only do `MergeGepUse` after `ReplaceUsesForLoweredUDTImpl` instead of before.